.env.sample

```diff
@@ -11,6 +11,21 @@
 export NODE_ENV=development
 export PORT=9000
 export ALLOW_HTTP=true
+export SUBDOMAIN=xxx
+export HOSTNAME=xxx
+export PROTOCOL=http
+export USERNAME_SELECTOR=xxx
+export PASSWD_SELECTOR=xxx
+export SIGN_IN_SELECTOR=xxx
+export AUTH_PATH=xxx/xxx
+export AUTH_USERNAME=xxx
+export AUTH_PASSWORD=xxx
+export ALLOW_UNKNOWN_BODY=true
+export HDFS_USER=root
+export HDFS_HOST=localhost
+export HDFS_PORT=50070
+export HDFS_PATH=/webhdfs/v1
```

Comment:
Maybe call it `WEBHDFS_URL` or something like that to avoid confusion with file path